modules/kml/src/gpx-loader.ts

```diff
@@ -15,7 +16,7 @@ const GPX_HEADER = `\
  * Loader for GPX (GPS exchange format)
  * @type {LoaderObject}
  */
-export const GPXLoader = {
+export const GPXLoader: LoaderObject = {
```

Comment:
Drop the jsdoc `@type`. You can seach the entire module for `@type` those should be removed in TS files.